src/components/Slider/__tests__/Slider.interactions.js

```diff
@@ -9,24 +9,56 @@ const changeSliderValueByClickingOnTrackTest = async canvas => {
   // prepare: take sizes of slider and waiting for render
   const elRail = canvas.getByTestId("monday-slider-show-value-s__rail");
   const rect = elRail.getBoundingClientRect();
-  const elThumb = await waitForElementVisible(() => within(elRail).getByRole("slider"));
+  const elThumb = await within(elRail).findByRole("slider");
```

Comment:
waitForElementVisible - add failed expectations at log :(
waitForElementVisible replaced by correct findByRole

![waitForElementVisible](https://user-images.githubusercontent.com/96241324/160640659-f3d1ee1d-f1da-4990-b61a-da5948091a02.png)

Reply:
good thing you noticed it. (behind the scenes `waitFor` is attempting to find the element multiple times, and each failure adds an error).
would you say that we should avoid `waitForElementVisible` to prevent such errors in the log?

Reply:
I think yes ...